android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -236,8 +244,11 @@ public void run() {
                 }
             }, 200);
 
-            startTimer(3000);
-
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
@sujitacharya2005 where do we remove the existing chrome permission dialog for app open ?